kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/ScanSuite.scala

```diff
@@ -888,9 +888,7 @@ class ScanSuite extends AnyFunSuite with TestUtils
            |) USING delta
            |TBLPROPERTIES(
            |'delta.dataSkippingStatsColumns' = 'c1,c2,c3,c4,c5,c6,c9,c10',
-           |'delta.columnMapping.mode' = 'name',
-           |'delta.minReaderVersion' = '2',
-           |'delta.minWriterVersion' = '5'
+           |'delta.columnMapping.mode' = 'name'
```

Comment:
This change is because now all the test features are enabled in delta-spark, it ends up adding `TestRemovableLegacyReaderWriterFeature` to the list. The table starts with protocol (2, 5) because of the these properties and due to timestampNtz, it upgrade the protocol and adds all implicitly enabled features as of (2,5) to the table (the test table feature is at 2, 5).